docs/pull-request-information.md

```diff
@@ -0,0 +1,25 @@
+## **Pull Request Instructions**
```

Comment:
for the pull request template , we have an open issue #106 , and the expectation is that it relates to github workflows so that a template is autogenerated whenever anyone tries to create a pr, perhaps let me add more context to the issue #106.

Reply:
#106 is a closed pull request!!!

Reply:
It was closed as complete...for the case of pull request template, try creating a pr and you'll see the template

Reply:
we could move the pull request info into `CONTRIBUTING.md`

Reply:
so i should copy the info and paste it into the `CONTRIBUTING.md`

Reply:
yah sure..pull request content is in regards to contributing